src/components/MessageWidget.scss

```
@@ -6,6 +6,16 @@
 }
 
 .message {
+  >* {
```

```suggestion
  > * {
```
好みもあると思うけどここはスペース入れたほうがよさそうかな